### PR TITLE
New release 2.1.8

### PR DESCRIPTION
**Pull request by mayanks**

### CODE REVIEW

File: gcs/CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ### 2.1.7 - 2022-02-03
```
Comment (arunkumarchacko) [inline gcs/CHANGES.md:1]: Should this be 2.1.8? Also, should we update the date?

Comment (mayanks) [reply gcs/CHANGES.md:1]: Thanks for pointing this out Arun

**Pull request merged**